docs/pages/next-steps/additional-resources.md

```diff
@@ -2,45 +2,39 @@
 title: Additional resources
 ---
 
-The following resources are useful for learning Expo and some of the projects that it depends on.
+The following resources are useful for learning about Expo tooling and services.
 
 ## Expo Blog
 
 - [Exposition](https://blog.expo.dev/) - our official blog, where we post release notes every month and other Expo related content at random intervals.
```

Comment:
```suggestion
- [Exposition](https://blog.expo.dev/) - Our official blog, where we post release notes every month and other Expo related content at random intervals.
```

Capitalizing the first letter of description to match the style of the rest of the document.